test/fixtures/recurring-application-charge/res/customize.json

```
@@ -0,0 +1,25 @@
+{
+    "recurring_application_charge": {
+        "id": 455696195,
+        "name": "Super Mega Plan",
+        "api_client_id": 755357713,
+        "price": "15.00",
+        "status": "active",
+        "return_url": "http://yourapp.com",
+        "billing_on": "2018-12-15",
+        "created_at": "2018-11-15T16:22:02-05:00",
+        "updated_at": "2018-11-15T16:23:29-05:00",
+        "test": null,
+        "activated_on": "2018-11-15",
+        "trial_ends_on": "2018-11-15",
+        "cancelled_on": null,
+        "trial_days": 0,
+        "capped_amount": "100.00",
```

```suggestion
        "capped_amount": "200.00",
```
Falls inline with the req amount

---

Interestingly, 200 in the request + 100 in response is what is part of the Shopify docs. I think the tests are built around these samples. You are right though - so I'll change